Makefile

```diff
@@ -8,7 +8,7 @@ lint:
 	@flake8 foursight_core || echo "'flake8 foursight_core' failed."
 	@flake8 tests || echo "'flake8 tests' failed."
 
-build:  # builds
+build:  react
```

Comment:
I poked around in doc for gnu `make` for a while but couldn't find a definite answer to this, maybe you know: Does this mean it will always still run but will first run react? Is there some advantage of this over what we have usually done in our scripts which is:
```
build:
	make react
	make configure
	poetry install
```